onmt/data/SampledDataset.lua

```diff
@@ -0,0 +1,266 @@
+--[[ Data management and batch creation. Handles data created by `preprocess.lua`. ]]
+require 'torchx'
```

Comment:
considering the size of AliasMultinomial.lua - we should rather integrate it in onmt as we used to integrate LinearNoBias - waiting for the module to pass into standard libraries